core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/speech/recognizer/manager/AbstractSpeechRecognizerManager.java

```diff
@@ -191,11 +187,6 @@ public void handleListeningState(boolean listening) {
 		 */
 	}
 
-	/**
-	 * Notify the System that sound is (not longer) Playing
-	 * T O D O private void notify(boolean listening) {}
```

Comment:
TODO for the Notify method for the Message-Bus
Maybe do not delete yet

`Notify the System that sound is (not longer) Playing` ->
`Notify the System that SR is (not longer) Listening`

Reply:
Opened Issue #203 for that.